### DISCUSSION TITLE
Plugin folder isn't create.

### DISCUSSION THREAD

**Discussion by KendoTarakate** [IzzelAliz/Arclight#1257] [General]
I start server on my computer but when I run Run.bat I create everything except for plugin folder. I tried to figure it out but not knowing how.
![Uploading image_2024-03-08_121429747.png…]()

**Comment by luccaalentino2** [comment]
Bro it doesn't appear to me either, did you manage to solve it?

**Comment by Darth23** [comment]
to create and run Arclight properly, just always open the Arclight jar.

**Comment by sod7m** [comment]
Create Start.bat with:
@echo off
java -Xmx6G -jar arclight-forge-1.20.1-1.0.5.jar nogui
pause

arclight-forge-1.20.1-1.0.5.jar my server file
After that all must work
